public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionComplete.js

```diff
@@ -140,6 +140,20 @@ function ModalMissionComplete (svl, missionContainer, taskContainer,
             svl.ui.leftColumn.confirmationCode.attr('data-toggle','popover');
             svl.ui.leftColumn.confirmationCode.attr('title','Submit this code for HIT verification on Amazon Mechanical Turk');
             svl.ui.leftColumn.confirmationCode.attr('data-content',svl.confirmationCode);
+
```

Comment:
Why is the code for hiding confirmation code popover repeated twice - once in audit.scala.html and once in this file?

Reply:
They handle 2 different cases:
 - whatever is on the audit view is to check if the turker has already completed the first mission when he reloads the audit page.
 - whatever is on the modalmissioncomplete js file is checked at the end of the actual first mission.

Reply:
If the code is exactly the same, can you modularize this code such that we don't have redundancy?

Reply:
I dont think its necessary. Its 2 lines that only look similar. One instance occurs before the svl.ui.leftColumn.confirmationCode ui element is initialized and the other occurs after. And only, we dont reuse them enough to warrant modularizing the code.

Reply:
ok